domains/_vercel.skibidi.json

```diff
@@ -4,6 +4,7 @@
     "discord": "skibidibagel"
   },
   "record": {
-    "TXT": "vc-domain-verify=bundler.skibidi.is-a.dev,0ed5e66158d45b6bd83d"
+    "TXT": "vc-domain-verify=bundler.skibidi.is-a.dev,0ed5e66158d45b6bd83d",
+    "TXT": "vc-domain-verify=skibidi.is-a.dev,b75173107a422d2f8244"
```

Comment:
Improper JSON. Should be one property an an array. See docs for more information.